website/content/docs/platform/k8s/vso/examples.mdx

```diff
@@ -11,6 +11,38 @@ The Operator project provides the following examples:
 - Sample use-cases are documented [here](https://github.com/hashicorp/vault-secrets-operator#samples)
 - A Terraform based demo can be found [here](https://github.com/hashicorp/vault-secrets-operator/tree/main/demo)
 
+## Configuring the JWT auth method to avoid API calls to the Kubernetes API Server
+
+Vault Secrets Operator supports using the [JWT auth method](/vault/docs/platform/k8s/vso/api-reference#vaultauthconfigjwt).
+In the default JWT configuration this involves an API call from the Vault server back to the Kubernetes API server to [verify](/vault/docs/auth/jwt#jwt-verification) the JWT.
+It is possible to configure Vault to use [`jwt_validation_pubkeys`](/vault/api-docs/auth/jwt#jwt_validation_pubkeys) fields of the JWT auth configuration
+to work in the Kubernetes environment and avoid the API call back to Kubernetes, improving performance of login requests,
+and removing the necessity to expose the Kubernetes API server externally.
+
+### Configure the JWT Auth method in Vault to use the JWKS keys from Kubernetes:
```

Comment:
I think it's valuable to create a pointer in the VSO docs, and include a brief motivating paragraph like you have above to explain how it's relevant to VSO specifically, but then from here the steps are not VSO-specific, and are documented pretty similarly here: https://developer.hashicorp.com/vault/docs/auth/jwt/oidc-providers/kubernetes#using-jwt-validation-public-keys. If that page is missing any finer details, I'd suggest polishing it up there and linking to it instead of repeating the steps in this page.

Reply:
Most excellent find on this link, I had no idea this existed, thanks!